java/src/jmri/jmrit/audio/AbstractAudioSource.java

```diff
@@ -84,6 +84,10 @@ public AbstractAudioSource(String systemName, String userName) {
         super(systemName, userName);
     }
 
+    boolean audioAlive() {
```

Comment:
Can you please make this public and rename to ```isAudioAlive()```? This would allow this method to be used by any class or scripts, and not just other members of this package.

Reply:
Will do so. Thank you very much!